idd/Energy+.idd.in

```diff
@@ -96204,69 +96204,11 @@ LifeCycleCost:UsePriceEscalation,
-   N17, \field Year 16 Escalation
-        \type real
-        \note The escalation in price of the energy or water use for the year expressed as a decimal.
-   N18, \field Year 17 Escalation
-        \type real
-        \note The escalation in price of the energy or water use for the year expressed as a decimal.
-   N19, \field Year 18 Escalation
-        \type real
-        \note The escalation in price of the energy or water use for the year expressed as a decimal.
-   N20, \field Year 19 Escalation
-        \type real
-        \note The escalation in price of the energy or water use for the year expressed as a decimal.
-   N21, \field Year 20 Escalation
-        \type real
-        \note The escalation in price of the energy or water use for the year expressed as a decimal.
-   N22, \field Year 21 Escalation
-        \type real
-        \note The escalation in price of the energy or water use for the year expressed as a decimal.
-   N23, \field Year 22 Escalation
-        \type real
-        \note The escalation in price of the energy or water use for the year expressed as a decimal.
-   N24, \field Year 23 Escalation
-        \type real
-        \note The escalation in price of the energy or water use for the year expressed as a decimal.
-   N25, \field Year 24 Escalation
-        \type real
-        \note The escalation in price of the energy or water use for the year expressed as a decimal.
-   N26, \field Year 25 Escalation
-        \type real
-        \note The escalation in price of the energy or water use for the year expressed as a decimal.
-   N27, \field Year 26 Escalation
-        \type real
-        \note The escalation in price of the energy or water use for the year expressed as a decimal.
-   N28, \field Year 27 Escalation
-        \type real
-        \note The escalation in price of the energy or water use for the year expressed as a decimal.
-   N29, \field Year 28 Escalation
-        \type real
-        \note The escalation in price of the energy or water use for the year expressed as a decimal.
-   N30, \field Year 29 Escalation
-        \type real
-        \note The escalation in price of the energy or water use for the year expressed as a decimal.
-   N31; \field Year 30 Escalation
-        \type real
-        \note The escalation in price of the energy or water use for the year expressed as a decimal.
+   N11,N12,N13,N14,N15,N16,N17,N18,N19,N20,N21,N22,N23,N24,N25,N26,N27,N28,N29,N30, \note Year (n-1) Escalation
+   N31,N32,N33,N34,N35,N36,N37,N38,N39,N40,N41,N42,N43,N44,N45,N46,N47,N48,N49,N50, \note Year (n-1) Escalation
+   N51,N52,N53,N54,N55,N56,N57,N58,N59,N60,N61,N62,N63,N64,N65,N66,N67,N68,N69,N70, \note Year (n-1) Escalation
+   N71,N72,N73,N74,N75,N76,N77,N78,N79,N80,N81,N82,N83,N84,N85,N86,N87,N88,N89,N90, \note Year (n-1) Escalation
+   N91,N92,N93,N94,N95,N96,N97,N98,N99,N100,N101; \note Year (n-1) Escalation
```

Comment:
@mbadams5 Can you confirm that this idd change is OK?  I just started looking over this PR and this is the first thing I noticed.  

@rraustad Like I said, I'm just now starting to look, so I might understand the need for this change shortly, but my current gut feeling is that there should be a way to include an extensible list here and have it work properly.  But maybe not.

Reply:
The problem with this is the IDEditor doesn't work when users have more than 30 years. Trane has been modifying the IDD all this time and requested that I make this change to save them that trouble.

Reply:
@rraustad Why did you delete the existing fully fleshed out fields up to year 30?  The IDF Editor problem will go away at some point.  As a compromise, keep the existing fields for years 1-30 , use shorthand for years 31-100 but *also make this extensible* to get a cleaner epJSON schema.  The input processing will throw out anything beyond 100 years.   @Myoldmopar @mbadams5